docs/develop/guides/working-with-momento-tokens.md

```diff
@@ -0,0 +1,54 @@
+
+:::info
+In addition to using an expiring auth token, it is best practice to store an auth token in a service like [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) or [GCP Secret Manager](https://cloud.google.com/secret-manager). Go to [Retrieving a Momento auth token from AWS Secrets Manager](/develop/sdks-integrations/aws-secrets-manager) for an example on using this.
+:::
+
+## Generating expiring tokens
+
+### Step 1: Sign up or log into the Momento console
+Go to the Momento Console and follow the instructions to login with your email address, Google account, or GitHub account.
+
+![image](/img/getting-started/console.png)
+
+### Step 2: Generate a Momento token
+In the console, open the menu and select token.
+
+![image of console menu](/img/getting-started/auth-token.gif)
+
+On the token page, select your 1/ cloud provider, 2/ an available region from the drop down list, 3/ when the token should expire, and then 4/ hit the "Generate Token" button.
+
+![image](/img/getting-started/select-provider-region.png)
+
+Scroll down and you will see your token in a grey box. The tokens in the screenshot has been blurred out, but yours won't be. The box contains 3 separate fields:
+1. Auth Token: this is the actual authentication token that gives your application access to Momento. This token will expired after the specified time period you selected.
+2. Refresh Token: if you wish to programmatically obtain a new auth token before the current one expires, you need this refresh token.
+3. Valid Until: this is purely informational, and displays the date and time the current auth token expires at.
+
+![image](/img/getting-started/generated-token.jpg)
+
+### Step 3: Store your Momento token
+If you wish to quickly test out Momento, click on the copy icon beside the auth token to copy the token to your clipboard and supply it to the Momento SDK. We recommend you also click the "Download JSON" button to store both the auth token and refresh token. Tokens are like passwords, the best practice is to store it in and retrieve it from a secure location such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/) or [GCP Secret Manager](https://cloud.google.com/secret-manager).
+
+### Step 4: Automating token refresh
+Once your application is running in production. You will need an automated script that periodically refreshes the auth token, **before** it expires. If you are running in AWS, an easy way to do this is by scheduling a function in AWS Lambda that does this for you. Check out our [1-click deploy example lambda](https://github.com/momentohq/auth-token-refresh-lambda) that you can use to automatically refresh your token in AWS Secrets Manager.
```

Comment:
I think somewhere we should mention that you also get a new refresh token, so that you can continue this refresh process indefinitely without needing any manual intervention.